tests/testing_tests/test_estimators.py

```diff
@@ -231,7 +231,7 @@ def test_program_11_2(self):
         self.assertEqual(round(model.params["Intercept"] + 90 * model.params["treatments"], 1), 216.9)
 
         # Increasing treatments from 90 to 100 should be the same as 10 times the unit ATE
-        self.assertEqual(round(model.params["treatments"], 1), round(ate, 1))
+        self.assertEqual(round(model.params["treatments"], 1), round(ate[0], 1))
```

Comment:
If we're returning pandas series now, would it be better to do an assert all rather than just grabbing the first one? Obviously it'll be the same result since the series are all one element long, but I think it would capture the vibe of what we're going for better